TOOLS/ce/src/main/scala/org.opalj.ce/CommentParser.scala

```diff
@@ -0,0 +1,308 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import java.nio.file.Path
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+import scala.io.Source
+import scala.util.Using
+import scala.util.control.Breaks.break
+import scala.util.control.Breaks.breakable
+
+/**
+ * The Comment Parser is responsible for parsing the HOCON files and its comments
+ * Do NOT use this class directly, use ConfigParserWrapper for safe parsing instead
+ */
+class CommentParser() {
+    var iterator: Iterator[String] = null
+    var line = ""
+
+    /**
+     * parseComments initiates the parsing process
+     * A ConfigNode can consist out of 3 possible types that are parsed differently: Objects, Lists and Entries
+     * The source node of a config file always is an object
+     * During parsing, the Parser will iterate the file and sort it into the ConfigNode structure.
+     * Since the Nodes can be nested and there can be multiple Nodes in one line, the Parser needs to examine most control structures like a stream and not linewise
+     * @param filePath accepts the path to a valid HOCON file
+     * @return returns the fully parsed file as a configObject
+     */
+    def parseFile(filePath: Path): ConfigObject = {
+        // Initialize iterator
+        println("Parsing :" + filePath.toString)
+
+        val lines =
+            Using(Source.fromFile(filePath.toString)) { source => source.getLines().toList }.getOrElse(List.empty[String])
+        iterator = lines.iterator
+
+        // Parse initial Comments
+        val initialComment = this.parseComments()
+        initialComment.addComment("@label " + filePath.toString.substring(filePath.toString.indexOf("opal") + 4))
```

Comment:
That is funny, because this is exactly what happened when I cloned my fork to directly push changes to this pull request.
Yes, it breaks.
But I already have a function in the file Locator that reliably finds out the root directory of opal, so I replaced it with that one.

Fixed.